tests/test_flow/fakes.py

```diff
@@ -194,7 +194,9 @@ def run_in_fake_gcp(fake_gcs: FakeGCS):
     mock_gcs_client.get_bucket = FakeBucket
 
     def create_aip_job(body, parent):
-        run_aip(body["trainingInput"]["args"][3])
+        # AIP logs are not printed locally.
+        with caplog.at_level(logging.CRITICAL):
```

Comment:
Why do we need to change the log level?

Reply:
Logs from running AIP do not show up in local instance, and we want to emulate it here. Added some comments.

Reply:
Ah, that makes sense -- thanks!